.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,3 +7,5 @@ Please double-check the following.
   - Did you write sufficient test code?
   - Did you write sufficient documentation?
 - Also take a look at [our compatibility policy](http://docs.chainer.org/en/stable/compatibility.html).
+
+Support is available on our [Slack Chat](https://bit.ly/chainer-slack).
```

Comment:
This is the pull request template. How about writing in the issue template instead?